src/main/resources/data-repository-openapi.yaml

```diff
@@ -653,30 +684,36 @@ definitions:
 
   JobModel:
     description: |
-      status of job
+      Status of job
     type: object
     required:
     - id
     - status
+    - submitted
     properties:
       id:
         $ref: '#/definitions/UniqueIdProperty'
+      description:
+        type: string
+        description: Description of the job's flight # from description flight input parameter
       status:
-        description: status of job
+        description: Status of job
         type: string
         enum:
-          - pending
           - running
           - succeeded
           - failed
-      description:
+      submitted:
```

Comment:
i think we need a separate status for failed and successfully cleaned up and failed and failed in undo (fatal?)

Reply:
Not sure that is worth reporting out. I would expect the status code to be 5xx and the error message to be enough. What would the client do with that info?

You do bring up something that we have not discussed at all: monitoring and alerting in Jade. If we get a dismal failure, we need to raise that (somehow) to dev ops so they know the software is broken and some action is needed.